apps/viewer/src/components/content.tsx

```diff
@@ -131,8 +131,14 @@ const Content = () => {
     const href = new URL(window.location.href);
     const streamName = href.searchParams.get('streamName') ?? import.meta.env.VITE_MILLICAST_STREAM_NAME;
     const streamAccountId = href.searchParams.get('streamAccountId') ?? import.meta.env.VITE_MILLICAST_STREAM_ID;
-    setupViewer(streamName, streamAccountId, projectingSourceId.current);
-    return stopViewer;
+    try {
+      setupViewer(streamName, streamAccountId, projectingSourceId.current);
+    } catch (err) {
```

Comment:
@vincentsong 

When the viewer app is opened with the below URL[just streamAccountID and not streamName], it still shows the Try Again Page.
https://dp-187--cosmic-dango-2cc8f7.netlify.app/?streamAccountId=KtpPTK
https://dp-187--cosmic-dango-2cc8f7.netlify.app/?streamAccountId=KtpPTK&streamName=





[10:37](https://dolbyio.slack.com/archives/D01PST55X4K/p1669937870646889)
https://dp-187--cosmic-dango-2cc8f7.netlify.app/?streamAccountId=KtpPTK&streamName=